src/machinetalk/config-service/configserver.py

```diff
@@ -236,15 +232,6 @@ def main():
     parser = argparse.ArgumentParser(
         description='Configserver is the entry point for Machinetalk based user interfaces'
     )
-    parser.add_argument(
```

Comment:
Attention: removing these arguments will break some configs. The name is also shown in QQV based application in the instances selection. Therefore, we shouldn't remove the option to name launchercmd independently.

E.g. how would it be possible to name an instance "Foo"? The only way would be to name all services "Foo", but then actual name announced by Avahi depends on pure luck.

Some applications filter for name rather than uuid for convenience.

Reply:
That's an interesting idea. 

What is the advantage of naming a service provided by an instance? 

A custom name for hostname does the same thing in human readable format, and when you run a manual mdns discovery the service name tells you what it does (since all service types are _machinekit.tcp presently).

Reply:
Also, does this apply to only configserver or to all of the published services using python?

Reply:
@machinekoder Bump for response. 

I can add a replacement token that allows passing in an instance name and substitutes it into the published announcement. I'm trying to get the announcements just more standardized, rather than created ad-hoc by every service that generates one. Presently, the hardcoded strings in every service (C side included) are messy for clients that *aren't* QQV. So, would the following be acceptable for existing configs and QQV:

```
(Prefix) (Service Name) on (FQDN).(Service Type).(Domain)
```

where prefix can be provided by a command line option? If not provided, we default to an INI specified prefix such as "MK". This would produce for launchercmd:

"Foo LauncherCmd on machinekit.local"

as the broadcast value. I'm pretty sure the end result would match identically what is being produced now by the hardcoded string in the service files, but it opens the door for us to have centralized formatting.